classes/services/PKPNavigationMenuService.inc.php

```diff
@@ -379,4 +331,29 @@ public function getMenuTree(&$navigationMenu) {
 			}
 		}
 	}
+
+	/**
+	 * Change NavigationMenuItem title according to template parameters if possible
+	 * @param $templateMgr \TemplateManager
+	 * @param $navigationMenu \NavigationMenu 
+	 */
+	public function transformNavMenuItemTitle($templateMgr, &$navigationMenuItem) {
+		// Transform an item title if the title includes a {$variable}
```

Comment:
Move this comment into the docblock above.

Reply:
OK